.github/workflows/versions-propagation-auto-merge.yml

```
@@ -2,13 +2,17 @@ name: Versions propagation auto-merge
 on:
   workflow_call:
 
+permissions:
+  pull-requests: write
+  contents: write
+
 jobs:
   enable-auto-merge:
     runs-on: ubuntu-latest
     if: ${{ github.actor == 'alfresco-build' && contains( github.event.pull_request.labels.*.name, 'updatebot') }}
     steps:
       - name: Enable auto-merge for propagation Pull Request
-        run: gh pr merge --auto --squash "$PR_URL"
+        run: gh pr merge --auto --rebase "$PR_URL"
         env:
           PR_URL: ${{github.event.pull_request.html_url}}
           # this token CANNOT be the default  `GITHUB_TOKEN`, otherwise the merge of the PR will not trigger a build
```

```suggestion
          # this token CANNOT be the default `GITHUB_TOKEN`, otherwise the merge of the PR will not trigger a build
```
(sorry, OCD kicking in :D)